sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/jms/properties/ServiceBusJmsProperties.java

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.cloud.autoconfigure.jms.properties;
+
+import org.springframework.boot.autoconfigure.jms.JmsPoolConnectionFactoryProperties;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.NestedConfigurationProperty;
+
+/**
+ * Configuration properties for Service Bus JMS.
+ */
+@ConfigurationProperties(prefix = ServiceBusJmsProperties.PREFIX)
+public class ServiceBusJmsProperties extends AzureServiceBusJmsProperties {
```

Comment:
We should reconsider of the class name to make it more meaningful.

Reply:
yes, I am considering it

Reply:
append the attribute to the AzureServiceBusJmsProperties.